### ISSUE TITLE
Idea: Mark tasks as completed

### ISSUE THREAD

**Issue by CodeWithDennis** [robertarnorsson/gzwmap#12]
It would be nice to show finished tasks on the map. Maybe we can switch the marker's icon to one with a check inside it. I suggest using local storage instead of making people create accounts.

**Comment by robertarnorsson** [comment]
Maybe also have a clear data and export and import data so people can sync there data

**Comment by CodeWithDennis** [comment]
![Scherm­afbeelding 2024-05-12 om 16 08 06](https://github.com/robertarnorsson/gzwmap/assets/23448484/bb49c848-882d-4a37-a6e1-f93e018f6b87)

Something very minimal like this would fit

**Comment by robertarnorsson** [comment]
Yeah and the market in the map could also maybe be grayed out or something
